skactiveml/classifier/_wrapper.py

```diff
@@ -322,3 +350,874 @@ def __getattr__(self, item):
             return getattr(self.estimator_, item)
         else:
             return getattr(self.estimator, item)
+
+
+class KernelFrequencyClassifier(ClassFrequencyEstimator):
+    """KernelFrequencyClassifier
+
+    Implementation of a wrapper class for skactiveml classifiers such that
+    `predict_freq` can be used by calculating the frequencies using other 
+    models (e.g., KernelDensity from scikit-learn or ParzenWindowClassifier).
+
+    Parameters
+    ----------
+    estimator : sklearn.base.SkactivemlClassifier with predict_proba method
+        scikit-learn classifier that is able to deal with missing labels.
+    classes : array-like of shape (n_classes,), default=None
+        Holds the label for each class. If none, the classes are determined
+        during the fit.
+    missing_label : scalar or string or np.nan or None, default=np.nan
+        Value to represent a missing label.
+    cost_matrix : array-like of shape (n_classes, n_classes)
+        Cost matrix with `cost_matrix[i,j]` indicating cost of predicting class
+        `classes[j]` for a sample of class `classes[i]`. Can be only set, if
+        `classes` is not none.
```

Comment:
add `use_only_marginal_frequencies` (evlt rename): sums up frequencies as the number of nearby labels and multiplies it by predict_proba from estimator.

Reply:
`use_estimator_proba`